ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -1,76 +1,410 @@
-                    onClick={() => window.open(supportUrls.createBillingAccount, '_blank')}>
-              Read Instructions
-            </Button>
-          </FlexColumn>
-        </ModalBody>
+export const CreateBillingAccountModal = ({onClose}: Props) => {
+  const {profile: {
+    contactEmail,
+    givenName,
+      familyName,
+      verifiedInstitutionalAffiliation,
+      username
+  }} = useStore(profileStore);
+  const [currentStep, setCurrentStep] = useState(0);
+  const [phoneNumber, setPhoneNumber] = useState();
+  const [invalidPhoneNumberInput, setInvalidPhoneNumberInput] = useState<boolean>(null);
+  const [useCreditCard, setUseCreditCard] = useState<boolean>(null);
+  const [nihFunded, setNihFunded] = useState<boolean>(null);
 
-        <hr style={{width: '100%', backgroundColor: colors.primary, borderWidth: '0px', height: '1px',
-          marginTop: '1rem',
-          marginBottom: '0.5rem'}}/>
+  const validatePhoneNumber = (phoneInput: string) => {
+    const sanitizedPhone = phoneInput.replace(/\D/g, '');
 
-        <ModalFooter style={{marginTop: 0, justifyContent: 'flex-start'}}>
-          <FlexColumn>
-            <TextColumn>
-              <p style={{marginTop: 0}}>Don't already have a Google Cloud account?</p>
-              <a href='https://cloud.google.com' target='_blank'>Create an account</a>
-            </TextColumn>
+    if (sanitizedPhone.length >= 10 && sanitizedPhone.length <= 11) {
+      setInvalidPhoneNumberInput(false);
+      setPhoneNumber(sanitizedPhone);
+    } else {
+      setInvalidPhoneNumberInput(true);
+    }
+  };
+
+  const sendCreateBillingEmail = fp.flow(
+    withErrorModal({
+      title: 'Failed To Send Email',
+      message: 'An error occurred trying to send email. Please try again.',
+    })
+  )(async() => {await profileApi().sendBillingSetupEmail(
+    {
+      phone: phoneNumber,
+      paymentMethod: useCreditCard ? BillingPaymentMethod.CREDITCARD : BillingPaymentMethod.PURCHASEORDER,
+      isNihFunded: nihFunded,
+      institution: verifiedInstitutionalAffiliation.institutionDisplayName
+    }
+  );
+    setCurrentStep(4);
+  });
 
-            <TextColumn>
-              <div>Add [ {this.props.profileState.profile.username} ] as a 'User' to your Google Cloud billing account.</div>
-              <a href='https://console.cloud.google.com/billing' target='_blank'>Add user</a>
-            </TextColumn>
+  return <Modal width={650} onRequestClose={() => onClose()}>
+        <ModalBody style={{marginTop: '-4'}}>
+          <FlexColumn>
+            <FlexRow style={{alignItems: 'center', width: '620px', marginBottom: '0.8rem'}}>
+              <GoogleCloudLogoSvg style={{height: '33px', width: '207px', marginLeft: '-0.5remx', marginRight: '0.5rem'}}/>
+              <div style={styles.line}></div>
+              <div style={{paddingTop: 5, marginLeft: '1rem', marginRight: '2rem'}}>
+                <div style={styles.textHeader}>Create billing account</div></div>
```

Comment:
nit current indentation is misleading: `</div>` should be on the next line, steps 1-3 shoudl be un-indented